Source/WebCore/crypto/cocoa/CryptoAlgorithmECDSAMac.cpp

```diff
@@ -31,37 +31,13 @@
 #include "CryptoAlgorithmEcdsaParams.h"
 #include "CryptoDigestAlgorithm.h"
 #include "CryptoKeyEC.h"
+#if HAVE(SWIFT_CPP_INTEROP)
+#include <pal/PALSwiftUtils.h>
+#endif
 
 namespace WebCore {
 #if HAVE(SWIFT_CPP_INTEROP)
 
-static PAL::HashFunction toCKHashFunction(CryptoAlgorithmIdentifier hash)
```

Comment:
I guess this was moved, huh?

Reply:
This was moved to a header to avoid de-duplication as now it is needed at 3 different places.